### PR TITLE
[BUGFIX] ensure outputPath contains only symlinks

### PR DESCRIPTION
**Pull request by stefanpenner**

### CODE REVIEW

File: lib/broccoli_sass_compiler.js [hunk 1]
```diff
@@ -667,9 +671,16 @@ module.exports = class BroccoliSassCompiler extends BroccoliPlugin {
...
+    let outputFile = details.fullCssFilename;
 
+    mkdirp.sync(path.dirname(cachedFile));
+    fs.writeFileSync(cachedFile, result.css);
+
+    mkdirp.sync(path.dirname(outputFile));
+    ensureSymlink(cachedFile, outputFile);
+
+    return this.events.emit("compiled", details, result);
     return this.events.emit("compiled", details, result);
```
Comment (mdebbar) [inline lib/broccoli_sass_compiler.js]: is it intentional to send the `compiled` event twice?

Comment (stefanpenner) [reply lib/broccoli_sass_compiler.js]: good catch, it is not.

**Comment by stefanpenner** [comment]
@rwjblue test added

**Comment by stefanpenner** [comment]
released as v4.1.1 🎉

**Comment by rwjblue** [comment]
Awesome, thank you @stefanpenner!

**Pull request merged**